packages/schematics/update/update/npm.ts

```diff
@@ -133,7 +133,14 @@ export function getNpmPackageJson(
     },
   );
 
-  const response = from<NpmRepositoryPackageJson>(resultPromise).pipe(shareReplay());
+  const response = from<NpmRepositoryPackageJson>(resultPromise).pipe(
+    shareReplay(),
+    catchError(err => {
+      logger.warn(err);
```

Comment:
Can you change this to `logger.warn(err.message || err)`?  The user doesn't need the full stack trace.